frontend/src/components/CreateEvent.js

```diff
@@ -10,4 +10,4 @@ const createevents = () => {
   )
 }
 
-export default createevents;
\ No newline at end of file
+export default createevents;
```

Comment:
please change variable name to createEvents to adhere to camelCase naming convention